go.mod

```diff
@@ -472,6 +472,8 @@ require (
 	sigs.k8s.io/yaml v1.4.0 // indirect; @grafana-app-platform-squad
 )
 
+require github.com/phpdave11/gofpdi v1.0.13
```

Comment:
Nit: Would be nice to put this in the main block - I guess it needs a codeowners too?

Reply:
Hmm not sure where that came from. Feel free to fix that